submissions/anthoniaAdollo/HTML/productPage.html

```diff
@@ -2,15 +2,16 @@
 <html>
 <head>
     <meta charset="UTF-8">
-    <title>Nurture-All About You!</title>
+    <title>Nurture-All About You! Our Products</title>
 </head>    
 <body>
     <nav>
-    <a href="#">Home</a>
-    <a href="#"> Product Categories</a>
-    <a href="#">Mommy&Me Blog</a>
-    <a href="#">Contact Us</a>
-    </nav>
+        <a href="#">Home</a>
+        <a href="/HTML/productPage.html">Products</a>
```

Comment:
You can simply do this `href="productPage.html"`. Because it is on the same folder